libpromises/dbm_lmdb.c

```diff
@@ -569,6 +593,17 @@ DBPriv *DBPrivOpenDB(const char *const dbpath, const dbid id)
         }
         goto err;
     }
+#ifndef __MINGW32__
```

Comment:
This should be done right after `LmdbEnvOpen()` or in the `err:` section to make sure it also happens if the database fails to be opened.

Reply:
Fixing in the 3.12.x version since that's the easiest to test and run through CI. Will come back and address this here and in master. Thanks! :)